srcpkgs/mspds/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'mspds'
+pkgname=mspds
+version=3.15.1.1
+revision=1
+archs="x86_64 i686"
+makedepends="boost-devel hidapi-devel"
+depends="hidapi"
+short_desc="Texas Instrumets MSP Debug Stack"
+maintainer="Benjamin Calderon <benj.calderon@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://www.ti.com/tool/MSPDS"
+distfiles="https://dr-download.ti.com/software-development/driver-or-library/MD-4vnqcP1Wk4/3.15.1.1/MSPDebugStack_OS_Package_3_15_1_1.zip"
+checksum=e3a59a98c43de7a92e5814d8c3304026165e6d2551e60acaca1f08c6b1a4bac8
+tags="tilib libmsp430"
+
+do_build() {
+	ln -s ${XBPS_CROSS_BASE}/usr/include/hidapi/hidapi.h ThirdParty/include/
+
+	make ${makejobs}
```

Comment:
This should use the `build_style=gnu-makefile` to include all our build flags/variables set correctly on the command line.

Reply:
Adding this adds a bunch of variables that I'd need to unset to make this package work. Take a look at this other [package template](https://github.com/void-linux/void-packages/blob/master/srcpkgs/msp430-toolchain/template) that also builds code from TI